model/URLSegmentFilter.php

```diff
@@ -32,6 +32,7 @@ class URLSegmentFilter extends Object {
 		'/\s|\+/u' => '-', // remove whitespace/plus
 		'/[_.]+/u' => '-', // underscores and dots to dashes
 		'/[^A-Za-z0-9\-]+/u' => '', // remove non-ASCII chars, only allow alphanumeric and dashes
+		'/\/+/u' => '', // remove forward slashes in case multibyte is allowed an the "remove non-ASCII chars" rule doesn't apply
```

Comment:
dat line length - can I be a pain and ask you to shorten it? Perhaps `// remove slashes if multibyte is allowed and the "remove non-ASCII chars" rule doesn't apply`? That gets it under 120 chars :)